### DISCUSSION TITLE
Tie up email account with metamask account?

### DISCUSSION THREAD

**Discussion by myanzik** [rahataid/tayaba#91] [General]
Right now we can use multiple metamask account while i am logged in through email. I think we should restrict that to single metamask account.

For example: if donor sign in to system, donor should only be able to use a registered metatmask account.

What do you think?

**Comment by myanzik** [comment]
@santosh-rumsan @raghav-rumsan @abiskar-sapkota  Any suggenstions?

**Comment by argahv** [comment]
I think it's a valid option to handle ownership of the account using wallet address.
